qml/MonitorMenuBar.qml

```diff
@@ -75,6 +77,13 @@ MenuBar {
             text: (panels.showLeftSidebar) ? "Hide Left sidebar" : "Show Left Sidebar"
             onTriggered: leftSidebarHidden()
         }
+        Action {
+            text: (inactive_visible) ? "Hide Inactive Entities" : "Show Inactive Entities"
+            onTriggered: {
+                inactive_visible = !inactive_visible
+                controller.change_inactive_visible()
```

Comment:
The first operation will change the inactive_visible to false/true and the controller will loss the control of this variable defined in QML. I'd keep the controller to be the one in charge of the inactive_visible property value. This could be done with a signal from C++ and a Connection in this file.

Reply:
For future versions